src/openrct2/Date.h

```diff
@@ -0,0 +1,58 @@
+#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
+/*****************************************************************************
+* OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+*
+* OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+* For more information, visit https://github.com/OpenRCT2/OpenRCT2
+*
+* OpenRCT2 is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* A full copy of the GNU General Public License can be found in licence.txt
+*****************************************************************************/
+#pragma endregion
+
+#pragma once
+
+#include "common.h"
+
+extern "C"
+{
+    #include "localisation/date.h"
+}
+
+namespace OpenRCT2
+{
+    /**
+     * Represents the current day, month and year in OpenRCT2.
+     */
+    struct Date final
```

Comment:
The obvious giveaway to me is having no public fields and being able to interact only via methods. On top of that, static member functions.